src/config.ts

```diff
@@ -32,6 +32,7 @@ export type ConditionConfig = {
   blockingLabels: string[],
   requiredBodyRegex: string | undefined
   blockingTitleRegex: string | undefined
+  pullRequestAuthorRole: string
```

Comment:
Seeing this among the other config options, I'd suggest. to call this `requiredAuthorRole` (or `requriedPullRequestAuthorRole`):

```suggestion
  requiredAuthorRole: string
```

This also results in the function being called `hasRequiredAuthorRole`, which makes more sense I. think. let me know what you think.

Reply:
:+1: